src/runtime/objmodel.cpp

```diff
@@ -490,6 +490,17 @@ BoxedDict* Box::getDict() {
 }
 
 Box* Box::getattr(const std::string& attr, GetattrRewriteArgs* rewrite_args) {
+
+    if (cls->tp_getattr) {
+        if (rewrite_args)
+            REWRITE_ABORTED("");
+
+        Box* rtn = cls->tp_getattr(this, const_cast<char*>(attr.c_str()));
+        if (rtn == NULL)
+            raiseExcHelper(AttributeError, "%s", attr.c_str());
+        return rtn;
+    }
+
```

Comment:
Do we need this in Box::getattr() as well?  I would have thought that by the time we get here we would have checked for a tp_getattr already...